app/app.module.ts

```diff
@@ -1,21 +1,20 @@
 import { NgModule, APP_INITIALIZER } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MaterialModule, MdIconRegistry } from '@angular/material';
-import { BrowserModule } from '@angular/platform-browser';
```

Comment:
Conflicts not resolved correctly here - Coding guidelines (order third party modules alphabetically by their module name and imports of our code alphabetically by name) aren't applied anymore.